core/frontend/src/views/Autopilot.vue

```diff
@@ -182,7 +182,7 @@ export default Vue.extend({
       if (!boardname) {
         return false
       }
-      return ['Navigator', 'SITL'].includes(boardname)
+      return ['Navigator', 'Navigator64', 'SITL'].includes(boardname)
     },
```

Comment:
yes we should get this from the backend in a better way

Reply:
This feels like scop-creeping though. since we don't have that information easily available anywhere and a new endpoint would be required in a backed.

Unless it is and I'm missing something?